vignettes/data_cleaning.Rmd

```diff
@@ -47,7 +47,7 @@ gads <- pisa
 
 ## Data structure
 
-The raw data in a `GADSdat` object are represented as raw, unlabeled values. They can be accessed via `GADSdat$dat`. For the extraction of data suitable for data analyses, see the `extractData2()` function.
+The raw data in a `GADSdat` object is represented as raw, unlabeled values. It can be accessed via `GADSdat$dat`. For the extraction of data suitable for data analyses, see the `extractData2()` function.
```

Comment:
"data" is plural. please leave this as it is

Reply:
I'll change it back, but apparently both is ok.